### ISSUE TITLE
Raycast extension saying Dashlane CLI is not installed error [Dashlane Vault] ...

### ISSUE THREAD

**Issue by young-prefect** [raycast/extensions#9318]
### Extension

https://www.raycast.com/tm.wrnr/dashlane-vault

### Description

Extension saying it "Dashlane CLI not installed" error when dcli is installed properly (see screenshots).

<img width="880" alt="Terminal dcli Installed" src="https://github.com/raycast/extensions/assets/76721156/e2be027c-f427-471c-b4a2-c7d1fe2b3b4b">
<img width="776" alt="Raycast dcli Error" src="https://github.com/raycast/extensions/assets/76721156/8d18facc-522b-40ce-9cc1-ad8905ff8848">

### Steps To Reproduce

1. Install Raycast extension
2. Install Dashlane CLI
3. Log into Dashlane account through CLI
4. Run Raycast dcli Sync command
5. Outputs "Dashlane CLI not installed"
6. Run Raycast dcli Search or Notes command
7. Outputs "Dashlane sync failed"

**Comment by tmwrnr** [comment]
Hi @young-prefect,

unfortunately, this problem is already known and described in #9236. 

**TLDR:** I cannot currently recreate this issue on my Mac, which makes it very difficult to fix the error.

As I can see from your Github profile, you are also a developer. Could you do me a favor and help me narrow down the error?

**Steps:** 
1. Clone this repository: https://github.com/raycast/extensions
2. Run `npm i` in the `extensions/dashlane-vault` directory
3. Open file `src/helper/install-check.ts`
4. Change the content of the file `src/helper/install-check.ts` (remove install check):
```diff
export async function checkIfInstalled<T extends () => void>(cb: T) {
-  const isAvailable = await checkIfCliIsInstalled();
-  if (!isAvailable) {
-    return showToast({
-      title: "Dashlane CLI is not installed",
-      style: Toast.Style.Failure,
-    });
-  }

  return cb();
}
```
5. Run `npm run dev` and execute the **Sync** command.

After these steps, I wonder what the result is. Either an error is thrown because the CLI cannot be called for some reason or the command works and something is broken in the `checkIfCliIsInstalled` function. Either way, I hope we're one step closer to fixing the error.

Thanks in advance!

**Comment by erics118** [comment]
The issue could be that the dashlane CLI is installed in a location that the Raycast extension can't find. A possible solution is to add a preference to manually set the dashlane CLI path

**Comment by tmwrnr** [comment]
Hi @erics118,

Thank you for your message! I noticed that the 1Password extension also does this, but I didn't understand why. I believe this will resolve the issue.

**Comment by tmwrnr** [comment]
Hi @young-prefect ,

I have released a new version that allows you to set the path for the Dashlane CLI. The [extension page](https://www.raycast.com/tm.wrnr/dashlane-vault) now includes a troubleshooting section on how to obtain the path. Please try this and let me know if it works for you.

**Comment by javier-kovacs** [comment]
Hi @tmwrnr 

This solved the issue. My dcli installation path is: /opt/homebrew/bin/dcli

Thank you so much for the solution.

**Comment by tmwrnr** [comment]
@raycastbot close this issue

**Comment by young-prefect** [comment]
Awesome. All set up. Thank you!
